src/components/preview/preview.jsx

```diff
@@ -57,6 +57,7 @@ function Preview({
             width: `${width}px`,
             height: `${height}px`,
             overflow: "hidden",
+            boxShadow: "0 0 1px #aaa",
```

Comment:
Maybe this should reuse the styling from https://github.com/os2display/display-admin-client/blob/553bf6c81f8c65f9e1d913a6fcab84e0119a471d/src/components/slide/preview/remote-component-wrapper.scss#L4